packages/trader/src/Modules/Account/Sections/Security/ConnectedApps/connected-apps.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+
+class ConnectedApps extends React.Component {
+    state = {
+        is_loading: true,
+    };
+    componentDidMount() {
+        let apps_list = this.props.fetchConnectedApps();
```

Comment:
usage of `app_list` variable seems reduntant, you can chain directly on the `this.props.fetchConnectedApps()`.